LICENSE.spdx

```diff
@@ -0,0 +1,20 @@
+SPDXVersion: SPDX-2.1
+DataLicense: CC0-1.0
+PackageName: tauri
+DataFormat: SPDXRef-1
+PackageSupplier: Organization: The Tauri Programme in the Commons Conservancy
+PackageHomePage: https://tauri.app
+PackageLicenseDeclared: Apache-2.0
+PackageLicenseDeclared: MIT
+PackageCopyrightText: 2022-2022, The Tauri Programme in the Commons Conservancy
+PackageSummary: <text>Tauri is a rust project that enables developers to make secure
+and small desktop applications using a web frontend.
+                </text>
+PackageComment: <text>The package includes the following libraries; see
+Relationship information.
+                </text>
+Created: 2019-05-20T09:00:00Z
+PackageDownloadLocation: git://github.com/tauri-apps/tauri
```

Comment:
do we need to change this URL @nothingismagick ?

Reply:
I think we have the same problem on **all** repos :( we've been just copy pasting it

Reply:
and there's also some places where it isn't being added to the npm/crates.io package

Reply:
yeah I fixed a couple of these repos yesterday but as I said someone with licensing knowledge should go through all repos.

Reply:
> and there's also some places where it isn't being added to the npm/crates.io package

That I didn't fixt

Reply:
Just checked 
tao, wry, win7-notifications, create-tauri-app, window-vibrancy and window-shadow and they all include this file in the publish packages